File: search_img.php
<?php
$query = "water%20purifier";
$url = 'https://pixabay.com/images/search/' . $query . '/?type=illustration';
$context = stream_context_create(['http' => ['header' => 'User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36']]);
$html = file_get_contents($url, false, $context);
preg_match_all('/https:\/\/cdn\.pixabay\.com\/photo\/[^\"]+(?:jpg|png)/i', $html, $matches);
print_r(array_slice($matches[0], 0, 5));
?>

File: search_img.py
import urllib.request, re

req = urllib.request.Request('https://pixabay.com/images/search/painting%20wall/?type=illustration', headers={'User-Agent': 'Mozilla/5.0'})
try:
    html = urllib.request.urlopen(req).read().decode('utf-8')
    links = re.findall(r'https://cdn\.pixabay\.com/photo/[^"]+(?:jpg|png)', html)
    print("Links found: ", links[:3])
except Exception as e:
    print(e)
